helm/values.yaml

```diff
@@ -40,32 +40,15 @@ deployer:
   serviceAccount:
     enabled: true
 
-cas-postgres:
-  patroni:
-    resources:
-      limits:
-        cpu: "210m"
-        memory: "512Mi"
-      requests:
-        cpu: "70m"
-        memory: "256Mi"
-    persistentVolume:
-      size: "300Mi"
-      storageClass: netapp-block-standard
-    replicaCount: 3
-    env:
-      ALLOW_NOSSL: true
-      USE_WALG_BACKUP: "true"
-      USE_WALG_RESTORE: "true"
-    walE:
-      enable: false # to be changed when we do backups
-      gcsBucket: ~
-      # kubernetesSecret should be gcp-{{ namespace }}-{{ gcs.bucketSuffix }}-service-account-key
-      kubernetesSecret: ~
-    # options below are required if walE.enable is true
-  namespace: ~
-  gcs:
-    bucketSuffix: ~
+spilo:
+  resources:
+  # default values spec'd from spilo-chart repo
```

Comment:
The spilo-chart repo does say to adjust the resources as necessary, so I would keep the ones you had above